catalog/opensearch/catalog-opensearch-source/src/main/java/ddf/catalog/source/opensearch/impl/OpenSearchParserImpl.java

```diff
@@ -341,34 +341,42 @@ private boolean hasParameter(String parameter, List<String> parameters) {
    *
    * @param lon latitude in decimal degrees (WGS-84)
    * @param lat longitude in decimal degrees (WGS-84)
-   * @param radius radius, in meters
+   * @param radius search radius, in meters
    * @return Array of bounding box coordinates in the following order: West South East North. Also
    *     described as minX, minY, maxX, maxY (where longitude is the X-axis, and latitude is the
    *     Y-axis).
    */
-  private double[] createBBoxFromPointRadius(double lon, double lat, double radius) {
+  private static double[] createBBoxFromPointRadius(
+      final double lon, final double lat, final double radius) {
     double minX;
     double minY;
     double maxX;
     double maxY;
 
-    double lonDifference = radius / (LAT_DEGREE_M * Math.cos(lat));
-    double latDifference = radius / LAT_DEGREE_M;
-    minX = lon - lonDifference;
-    if (minX < MIN_LON) {
-      minX += MAX_ROTATION;
-    }
-    maxX = lon + lonDifference;
-    if (maxX > MAX_LON) {
-      maxX -= MAX_ROTATION;
-    }
+    final double latDifference = Math.toDegrees(radius / RADIUS_OF_THE_EARTH_IN_METERS);
```

Comment:
This method takes in WGS-84 coordinates but then makes the calculation treating the earth as a sphere. How much to we care about our precision here?

Reply:
I haven't thought about this. I'll do some more research and test with both the 2D and 3D maps.

Reply:
There is a disclaimer in the documentation:
https://github.com/codice/ddf/blob/2c280fca47752d4f0fe573827e815def461fd02d/distribution/docs/src/main/resources/content/_tables/OpenSearchSource.adoc#L61-L64

Reply:
Good enough for me.